### ISSUE TITLE
No cofactor with the asinh?

### ISSUE THREAD

**Issue by SamGG** [ebecht/DR_benchmark#1]
Hi,
Thanks for this interesting improvement of dimension reduction and sharing the code.
I noticed that there is no cofactor when transforming the CyTOF data of Samusik (I didn't check the others). Any reason for that?
https://github.com/ebecht/DR_benchmark/blob/245d74eb4e21325d7cf6240ad19ed0e4626b82ba/Samusik/Samusik_01_data_load_and_DR.R#L17
Best,
Samuel

**Comment by ebecht** [comment]
Hi @SamGG,

I did not use a co-factor in asinh for the first ~2 years I have worked with CyTOF data. Now I realize that a co-factor is a good way to lower the emphasis on "background" expression data. For most application it won't be too critical though (dimensionality reduction and clustering are mostly unaffected in my experience - two parameters density plots are however much nicer when using a co-factor).

Best,
Etienne
